### ISSUE TITLE
General styling of the page and layout

### ISSUE THREAD

**Issue by yharaskrik** [yharaskrik/mtg-visual-deckbuilder#5]

**Comment by huguenot412** [comment]
The top bar is pretty empty except for the hamburger menu icon. Perhaps we could fill it by moving the deck name there instead of / in addition to in the sidebar

**Comment by yharaskrik** [comment]
Good call, I also see that has where the navigation to other pages could be? We use angular material at trellis but am open to other ideas for sidenavs

**Comment by huguenot412** [comment]
I tried making some of the button material buttons but it wasn't taking the themed styling. Do you know if a them has been set up yet?

**Comment by yharaskrik** [comment]
Naw, we haven't added that at all yet, just need to `ng add @angular/material` and we should be good to go. I am also open to using something else like tailwind if you want, I have heard it is awesome.

**Comment by huguenot412** [comment]
I'll let you make the call on which way to go with that. I use Tailwind at work and I like it, but it's a much different approach since most of your styling is done through utility classes in the html. You only get access to the pre-built components if you pay for TailwindUI and even then any JS and animations needs to be hand-coded. I'd almost prefer to use material since we get more "out of the box" with it, plus I've been looking for an excuse to get more familiar with Material, but again I'm happy to follow your lead.

**Comment by yharaskrik** [comment]
Great! Material it is! I personally love material.

**Comment by huguenot412** [comment]
![image](https://user-images.githubusercontent.com/17086073/120242450-1b710380-c222-11eb-865a-f0d7ce143581.png)
Set up the theme. Any preference for which pre-built theme to use, or do you want to make a custom theme?
What will go inside the mana value boxes?

**Comment by yharaskrik** [comment]
I am fine with a prebuilt one for now. We can edit it later. 

Those mana value boxes should go inside the playmat area as those are the "columns" the cards will be in.

**Comment by huguenot412** [comment]
@yharaskrik I've been busy at work this past week and I'll be out of town through Monday so I won't be able to work on this much until next week. Hoping I'll have more time to pitch in once I get back. It's looking great!

**Comment by yharaskrik** [comment]
No worries at all @huguenot412! Enjoy the time off!
